src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -314,7 +321,7 @@
 
 helps['storage directory exists'] = """
     type: command
-    short-summary: Indicates whether the directory exists.
+    short-summary: Determines if a directory exists.
```

Comment:
There seems to be a lot of variety for these "exists" commands. Can we standardize them?